src/pages/workshop/appmod/index.mdx

```diff
@@ -36,21 +36,21 @@ import Globals from 'gatsby-theme-carbon/src/templates/Globals';
 
 1. Instructor will provide the following info:
     - OpenShift Console URL (OCP_CONSOLE_URL)
-    - The username and password for OpenShift and Git Server (default values are user1, user2, etc.. for users and `password` for password).
+    - The username and password for OpenShift and Git Server (default values are user01, user02, etc.. for users and `password` for password).
 
-1. Set `TOOLKIT_USERNAME` environment variable replace `user1` with assigned usernames
+1. Set `TOOLKIT_USERNAME` environment variable, replace `userXX` with assigned username (ex. `user01`)
     ```bash
-    TOOLKIT_USERNAME=user1
+    TOOLKIT_USERNAME=userXX
```

Comment:
I think we want this to be `user01` this way user can just copy and paster without modification if they are using the openlabs where he/she is the only user.